topdown/eval.go

```diff
@@ -323,7 +330,10 @@ func (e *eval) evalExpr(iter evalIterator) error {
 				return err
 			}
 			if !e.findOne {
```

Comment:
So if we cannot do EE for this expression, (`!e.findOne`), we'll either
1. suppress EE (`e.suppressEarlyExit`), or
2. shovel along the outer EE sentinel error

In words, how would one decide whether (1.) or (2.) is to be the case?